app/api/chewbacca/generateSummaryOfQualifications/route.ts

```diff
@@ -123,15 +125,26 @@ async function generateKeywordsFromRequirements(
   requirementText: string,
   competencies: string[],
 ): Promise<RequirementCompetency> {
-  const prompt = `Jeg har noen nøkkelord som systemet mitt støtter. Disse er ${competencies} svar hvilken nøkkelord som er relevant for kravet : "${requirementText}". Svaret skal kun inneholde ordene komma-separert`;
-  const response = await requestOpenai([{ role: "user", content: prompt }]);
+  const example: ChatCompletionRequestMessage[] = [
+    {
+      role: "user",
+      content:
+        'Jeg har noen nøkkelord som systemet mitt støtter. Disse er JIRA,KUBERNETES,AZURE,GCP,AWS,JAVA,C# svar hvilken nøkkelord som er relevant for kravet : "Kandidaten må være god på utvikling av skytjenester ". Svaret skal ikke inneholde noe annet enn kommaseparert liste med nøkkelord',
```

Comment:
,JAVA,C# svar hvilke -> ,JAVA og C#. Svar hvilke

Reply:
"og" er ikke nødvendig for da må jeg ta lista å legge en "og" på den og.  GPT skjønner begge semantikker. enig i at jeg bytter til "hvilke" da dette er riktig og sparer tegn samtidig.